src/Spryker/Zed/CodeGenerator/Business/GeneratorProjectTypeResolver/AbstractGeneratorProjectTypeResolver.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Zed\CodeGenerator\Business\GeneratorProjectTypeResolver;
+
+use Spryker\Zed\CodeGenerator\CodeGeneratorConfig;
+
+abstract class AbstractGeneratorProjectTypeResolver implements GeneratorProjectTypeResolverInterface
```

Comment:
Why do we need this resolver? Why not directly passing in the config and using the method?
This proxy seems to be overkill given the current usage.

Reply:
Maybe in future will be added more complex logic of resolving project types.